src/plugins/intel_npu/src/plugin/npuw/weights_bank.cpp

```diff
@@ -0,0 +1,5 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "weights_bank.hpp"
```

Comment:
Need to move some implementation here

Reply:
This is not resolved as far as I can tell

Reply:
Done